flixel/math/FlxVector.hx

```diff
@@ -10,6 +10,13 @@ class FlxVector extends FlxPoint
 	public static inline var EPSILON:Float = 0.0000001;
 	public static inline var EPSILON_SQUARED:Float = EPSILON * EPSILON;
 	
+	/**
+	 * Pooling mechanism to minimimze runtime allocations.
+	 * 
+	 * TODO: We need more evidence to decide whether liberal use of pooled objects
```

Comment:
Putting notes like this in public docs doesn't seem like a good idea to me, doesn't exactly induce confidence in API useres.